fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1695,6 +1695,24 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true, masterOnly = true)
     public static int cbo_default_sample_percentage = 10;
 
+    /*
+     * if true, statistics will be updated automatically
+     */
+    @ConfField(mutable = true, masterOnly = true)
+    public static boolean enable_auto_collect_statistics = false;
+
+    /*
+     * collect statistics periodically can be stressful on the cluster
+     */
+    @ConfField(mutable = true, masterOnly = true)
+    public static boolean enable_period_collect_statistics = false;
```

Comment:
What's diff between enable_period_collect_statistics and enable_auto_collect_statistics, besides these two options isn't used anywhere